browser/index.hbs

```diff
@@ -88,9 +96,16 @@
         </p>
         <p>
             PromoCode product code:
-            <input class="buy-promo-code-product-id" type="text" value="{{config.paymentPromoCodeProduct}}" />
+            <input class="product-id" type="text" value="{{config.paymentPromoCodeProduct}}" />
             <button class="buy-promo-code-product">Buy PromoCode Product</button>
         </p>
+    {{#if config.alternativeClient.paymentPromoCodeProduct}}
+        <p>
+            PromoCode product code for alternative client:
```

Comment:
why not instead of using `alternative` client use variable from config (`ALTERNATIVE_CLIENT_PAYMENT_PUBLISHER`) and then you will see that this is for in ex. `vg` publisher